control/controlclient/auto.go

```diff
@@ -767,6 +767,11 @@ func (c *Auto) UpdateEndpoints(endpoints []tailcfg.Endpoint) {
 	}
 }
 
+func (c *Auto) SetDiscoPublicKey(key key.DiscoPublic) {
```

Comment:
docs? (it's part of an interface, but the interface docs weren't accurate... this does an updateControl, which should be mentioned)